js-exercises/minima/minima.js

```diff
@@ -0,0 +1,28 @@
+const sort = array => array.sort((a, b) => a - b);
+
+const minima = (numberOfElements, inputArray) => {
+  if (!Array.isArray(inputArray)) {
+    throw Error(`Expected Array Got ${typeof inputArray}`);
+  }
+
+  if (inputArray.length === 0) {
+    throw Error('Got empty array as an input');
+  }
+
+  if (inputArray.length < numberOfElements) {
+    throw Error('Number of Elements to be returned is more then the array length.');
+  }
+
+  if (typeof numberOfElements !== 'number') {
+    throw Error(`Expected numberOfElements argument as number. Got ${typeof numberOfElements}`);
+  }
+
+  const sortedArray = sort(inputArray);
+  const slicedArray = sortedArray.slice(0, numberOfElements);
+  return slicedArray;
```

Comment:
it's not a big line to read, it's okay if it's a single line.

```suggestion
  return sortedArray.slice(0, numberOfElements);
```